cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -159,7 +160,10 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
     ).convert { name ->
         allPackageManagersByName[name]
             ?: throw BadParameterValue("Package managers must be one or more of ${allPackageManagersByName.keys}.")
-    }.split(",")
+    }.split(",").deprecated(
+        message = "--package-managers is deprecated, use -P ort.analyzer.enabledPackageManagers=[] instead.",
```

Comment:
Is `[]` correct here? Don't we expect a comma-separated list without surrounding `[]` on the command line?